### PR TITLE
Playlist Aggregation Party Users2

### PR DESCRIPTION
**Pull request by Mackenzie-Frey**
This branch implements functionality for a playlist to be created based on the party animals in a party. It works for any number of party users, including just the admin being in the party.
The algorithm makes an api call to get the top 5 most played artists for a user over the last 6 months. This is done for each party animal. The artist ids for all party animals are then combined together, checked for duplicates, and five artist ids are randomly selected. These five artist ids are then saved the party in the database and updated each time a new party animal joins the party. 

The api calls were measured and optimized by storing the seed artists for a user to the database.

Skipped test is in progress which will document the method the SDK player will call.

*** Please keep the comments.

### CODE REVIEW

**Comment by Mackenzie-Frey** [comment]
Closing PR to continue working on the branch.

**Comment by Mackenzie-Frey** [comment]
Reopneing pull request.

File: app/models/playlist.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  def select_seeds(all_ids)
+    all_ids.map do |party_user_ids|
+      party_user_ids.split(",")
+    end.flatten.uniq.sample(5).join(',')
+  end
+
+  def user_top_plays(party_user)
+    if party_user.seed_artists.nil?
+      party_user.update!(seed_artists:
+      SpotifyService.new(party_user.access_token).top_plays)
```
Comment (peregrinebalas) [inline app/models/playlist.rb]: We refactored this service to take a user, how much do we need to change?

Comment (peregrinebalas) [inline app/models/playlist.rb]: Let's change this :)

**Review by peregrinebalas — DISMISSED** [review]

**Comment by manojpanta** [comment]
Reviewed as a Team, Resolved conflicts!!

**Pull request merged**